highfidelity/hifi
# Case 21861: Mac Audio Push To Talk fix

# TEST PLAN
- Get on a Mac
- Launch interface in desktop mode
- Set push to talk enabled in audio app through `Settings > Audio`
- relaunch interface in desktop mode
- mic bar in top left should display "MUTED PTT-(T)" when T is not pressed, and unmute when T is pressed

- Get on a Windows PC
- Launch interface in desktop mode
- Set push to talk enabled in audio app through `Settings > Audio`
- relaunch interface in desktop mode
- mic bar in top left should display "MUTED PTT-(T)" when T is not pressed, and unmute when T is pressed
- switch to HMD mode
- mic bar in top left should display "MUTED PTT" when both grip buttons are not pressed, and unmute when both grip buttons are pressed

https://highfidelity.manuscript.com/f/cases/21861/Mac-Audio-overlay-shows-Muted-after-user-closes-and-reopens-Interface-with-PTT-enabled
